lib/model/model.go

```diff
@@ -1559,18 +1559,38 @@ func sendIndexTo(minSequence int64, conn protocol.Connection, folder string, fs
 	return maxSequence, err
 }
 
+func (m *Model) makeUsLastModified(fs []protocol.FileInfo) []protocol.FileInfo {
+	fsMod := make([]protocol.FileInfo, 0, len(fs))
+	for _, file := range fs {
+		file.ModifiedBy = uint64(m.id.Short())
```

Comment:
I think it would be cleaner to set this where we create the new FileInfo in walk.go instead?